### PR TITLE
[Manila] Changes to support Xena release

### PR DESCRIPTION
**Pull request by kpsap**
Based on reading from
https://docs.openstack.org/releasenotes/manila/wallaby.html and https://docs.openstack.org/releasenotes/manila/xena.html

@galkindmitrii  check

### CODE REVIEW

File: openstack/manila/templates/etc-configmap.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ data:
 {{ include (print .Template.BasePath "/etc/_manila.conf.tpl") . | indent 4 }}
   api-paste.ini: |
 {{ include (print .Template.BasePath "/etc/_api-paste.ini.tpl") . | indent 4 }}
-  policy.json: |
+  policy.yaml: |
 {{ include (print .Template.BasePath "/etc/_manila-policy.json.tpl") . | indent 4 }}
```
Comment (galkindmitrii) [inline openstack/manila/templates/etc-configmap.yaml:18]: manila-policy template should be renamed and converted to yaml
+ there seem to be a couple of changes in the policies according to: https://docs.openstack.org/releasenotes/manila/de/wallaby.html

**Comment by Carthaca** [comment]
The policy changes have been done in https://github.com/sapcc/helm-charts/pull/2934, but the changes to the `manila.conf` are still relevant, right?

**Comment by kpsap** [comment]
yes, this PR was created first and then realised that policy.yaml is acceptable in victoria as well, so create another PR for just policy.yaml. After 2934 merged, only quota will go for xena.

**Comment by kpsap** [comment]
since we are not using quota from manila, we can merge PR after Xena rollout just to avoid lot of warnings from various pods.

**Comment by Carthaca** [comment]
Ok, it is only a deprecation warning, so the config setting should still work the old way on xena, too

But it really is important for us, we explicitly want to set all quota to 0 by default for all resources

**Review by Carthaca — APPROVED** [review]

**Pull request merged**